[DOC] Adapt coc. (#50)
seqan/sharg-parser: CODE_OF_CONDUCT.md

# Code of Conduct {#about_code_of_conduct}

> The Sharg-parser adheres to the [Berlin Code of Conduct](https://berlincodeofconduct.org/). As the Sharg-parser was created and is maintained by the SeqAn team, it is part of the SeqAn project. Therefore, SeqAn's Code of Conduct applies to the Sharg-parser as well. Please, see [SeqAn's Code of Conduct](http://docs.seqan.de/seqan/3-master-user/about_code_of_conduct.html) for more information or how to enforce our Code of Conduct.

Reviewer: ```suggestion
The Sharg-parser adheres to the [Berlin Code of Conduct](https://berlincodeofconduct.org/). As the Sharg-parser was
created and is maintained by the SeqAn team, it is part of the SeqAn project. Therefore, SeqAn's Code of Conduct 
applies to the Sharg-parser as well. Please see 
[SeqAn's Code of Conduct](http://docs.seqan.de/seqan/3-master-user/about_code_of_conduct.html) for more 
information on our Code of Conduct.
```
